_episodes/08-conflict.md

```diff
@@ -0,0 +1,338 @@
+I'll be able to get 40 extra minutes of beauty rest
+=======
+Yeti will appreciate the cold
+>>>>>>> dabb4c8c450e8475aee9b14b4383acc99f42af1d
+~~~
+{: .output}
+
+Our change—the one in `HEAD`—is preceded by `<<<<<<<`.
+Git has then inserted `=======` as a separator between the conflicting changes
+and marked the end of the content downloaded from GitHub with `>>>>>>>`.
+(The string of letters and digits after that marker
+identifies the commit we've just downloaded.)
+
+It is now up to us to edit this file to remove these markers
+and reconcile the changes.
+We can do anything we want: keep the change made in the local repository, keep
+the change made in the remote repository, write something new to replace both,
+or get rid of the change entirely.
+Let's replace both so that the file looks like this:
+
+~~~
+$ cat mars.txt
+~~~
+{: .bash}
+
+~~~
+Cold and dry, but everything is my favorite color
+The two moons may be a problem for Wolfman
+But the Mummy will appreciate the lack of humidity
+We removed the conflict on this line
+~~~
+{: .output}
+
+To finish merging,
+we add `mars.txt` to the changes being made by the merge
+and then commit:
+
+~~~
+$ git add mars.txt
+$ git status
+~~~
+{: .bash}
+
+~~~
+On branch master
+All conflicts fixed but you are still merging.
+  (use "git commit" to conclude merge)
+
+Changes to be committed:
+
+	modified:   mars.txt
+
+~~~
+{: .output}
+
+~~~
+$ git commit -m "Merge changes from marsTemp"
+~~~
+{: .bash}
+
+~~~
+[master 2abf2b1] Merge changes from marsTemp
+~~~
+{: .output}
+
+Git keeps track of what we've merged with what,
+so we don't have to fix things by hand again. If
+we make another change to the `marsTemp` branch
+
+~~~
+$ git checkout marsTemp
+$ echo "The polar caps will probably be Yeti's home" >> mars.txt
+$ git add mars.txt
+$ git commit -m "A note about Yeti's home"
+~~~
+{: .bash}
+
+~~~
+[master 34avo82] A note about Yeti's home
+ 1 file changed, 1 insertion(+)
+~~~
+{: .output}
+
+And merge that change into master
+
+~~~
+$ git checkout master
+$ git merge marsTemp
+~~~
+
+~~~
+Updating 12687f6..x792csa1
+Fast-forward
+ mars.txt | 1 +
+ 1 file changed, 1 insertions(+), 0 deletions(-)
+~~~
+{: .output}
+
+There is no conflict and our changes are added automatically
+
+~~~
+$ cat mars.txt
+~~~
+{: .bash}
+
+~~~
+Cold and dry, but everything is my favorite color
+The two moons may be a problem for Wolfman
+But the Mummy will appreciate the lack of humidity
+We removed the conflict on this line
+The polar caps will probably be Yeti's home
+~~~
+{: .output}
+
+We don't need to merge again because Git knows someone has already done that.
+
+Git's ability to resolve conflicts is very useful, but conflict resolution
+costs time and effort, and can introduce errors if conflicts are not resolved
+correctly. If you find yourself resolving a lot of conflicts in a project,
+consider one of these approaches to reducing them:
+
+- Try breaking large files apart into smaller files so that it is less
```

Comment:
These suggestions seem to be written for collaborating conflicts.  Maybe you want to change them to suggestions for working on your own.  Or add a little bit of background that you might expect conflicts when working with collaborators and these are things you can do to avoid conflicts? or both?